### PR TITLE
Issue #25: ordered_importS is case sensitive since Coder 8.3.25.

### LINKED ISSUE #25: Coder 8.3.25: sort import
## Steps required to reproduce the problem

In https://www.drupal.org/project/coder/releases/8.3.25

Import sorting is now case sensitive: https://www.drupal.org/node/3470716

### PR DESCRIPTION
**Pull request by FlorentTorregrosa**
This PR

Fixes #25.

### CODE REVIEW

**Review by drupol — APPROVED** [review]

File: tests/Unit/Config/Drupal8Test.php [hunk 1]
```diff
@@ -313,6 +313,7 @@ public function testConfigDefault(): void
                 'sort_algorithm' => 'alpha',
             ],
             'ordered_imports' => [
+                'case_sensitive' => TRUE,
```
Comment (WengerK) [inline tests/Unit/Config/Drupal8Test.php:316]: just fix case here as GrumPHP says in CI then would merge.

Comment (WengerK) [reply tests/Unit/Config/Drupal8Test.php:316]:
```suggestion
                'case_sensitive' => true,
```

Comment (FlorentTorregrosa) [reply tests/Unit/Config/Drupal8Test.php]: Change done.

**Comment by WengerK** [comment]
Seems still having issue with phpcs x)

1) src/Fixer/BlankLineBeforeEndOfClass.php (array_indentation, no_extra_blank_lines, blank_line_between_import_groups)

**Comment by FlorentTorregrosa** [comment]
Ok but the src/Fixer/BlankLineBeforeEndOfClass.php is unaffected by this PR. Should it be fixed in this PR or a dedicated one?

**Comment by WengerK** [comment]
I'll fix tests on https://github.com/drupol/phpcsfixer-configs-drupal/pull/27 and rebase this proposal PR

**Comment by WengerK** [comment]
rebased PR on #29

**Pull request closed without merging**